### PR TITLE
See if the reverse proxy will add the nonce

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

gids is causing some CSP errors:

image

If this script tag had the right nonce on it, it shouldn't cause any errors.
image

gids is behind our reverse proxy, and in VFS apps the reverse proxy looks for strings matching `**CSP_NONCE**` and does a string substitution with the true nonce value present in the CSP header.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]

**Comment by zurbergram** [comment]
This is related to https://github.com/department-of-veterans-affairs/va.gov-team/issues/8910

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
   <title>GibctDataService</title>
   <%= stylesheet_link_tag    'application', media: 'all' %>
-  <%= javascript_include_tag 'application' %>
+  <%= javascript_include_tag 'application', nonce: '**CSP_NONCE**' %>
```
Comment (rianfowler) [inline app/views/layouts/application.html.erb:6]: [This](https://github.com/department-of-veterans-affairs/devops/blob/96861837aead268e79938695d5b839bd466f2514/ansible/deployment/config/revproxy-vagov/templates/nginx_revproxy.conf.j2#L328) is where we're doing the content swap in the reverse proxy.

**Review by rianfowler — APPROVED** [review]
lgtm- what's the plan for testing this? who is going to validate it and where (staging or prod)?

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (LindseySaari) [reply app/views/layouts/application.html.erb:6]: Can you point to the substitution logic?

Comment (LindseySaari) [reply app/views/layouts/application.html.erb:6]: I would possibly coordinate with devops to see if there's a workaround for this. I'm seeing that the substitution takes place in the prod content security policy yml with some additional logic in the ngnix_revproxy config in the devops repo. An ops engineer may be more familiar

Comment (bkjohnson) [reply app/views/layouts/application.html.erb:6]: I've talked with @zurbergram about this and once this is merged I will check staging to see if we are still seeing that CSP error in the browser console.

Comment (johnpaulashenfelter) [reply app/views/layouts/application.html.erb:6]: Checked w/ @rianfowler and he's on board with that plan. This is a lot more front-end oriented than most of these PRs so a few more steps in the review chain

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**